tests/console/yast2_samba.pm

```diff
@@ -329,7 +341,7 @@ sub run {
         record_soft_failure "bsc#1088152";
         setup_yast2_ldap_server;
     }
-    setup_samba;
+    setup_samba($self);
```

Comment:
You could use `$self->setup_samba` here which is more common, but still is a matter of taste.